tests/dag_processing/test_processor.py

```diff
@@ -786,14 +786,25 @@ def test_import_error_tracebacks(self, tmpdir):
             assert len(import_errors) == 1
             import_error = import_errors[0]
             assert import_error.filename == unparseable_filename
-            expected_stacktrace = (
-                "Traceback (most recent call last):\n"
-                '  File "{}", line 3, in <module>\n'
-                "    something()\n"
-                '  File "{}", line 2, in something\n'
-                "    return airflow_DAG\n"
-                "NameError: name 'airflow_DAG' is not defined\n"
-            )
+            if PY311:
```

Comment:
We may want to use >=3.11 instead of only include 3.11? At least 3.12 will use the same format, if I remember correctly.